ui/pages/home/home.component.js

```diff
@@ -345,8 +346,49 @@ export default class Home extends PureComponent {
       setWaitForConfirmDeepLinkDialog(false);
     });
     ///: END:ONLY_INCLUDE_IF
+
+    this.triggerNewPrivacyPolicyToast();
   }
 
+  triggerNewPrivacyPolicyToast() {
+    const newPrivacyPolicyDate = new Date(2024, 5, 6);
+
+    const userHasClosedToast = Boolean(
+      window.localStorage?.getItem('privacyPolicyToastClosed'),
```

Comment:
We don't use `localStorage` in this way; check out my original Survey PR to see how I store `surveyLinkLastClickedOrClosed` in `appState` -- that's how we can track whether the user has seen/acknowledged it or not

https://github.com/MetaMask/metamask-extension/pull/21918